iocore/eventsystem/I_IOBuffer.h

```diff
@@ -58,6 +58,7 @@ class VIO;
 inkcoreapi extern int64_t max_iobuffer_size;
 extern int64_t default_small_iobuffer_size;
 extern int64_t default_large_iobuffer_size; // matched to size of OS buffers
+extern int iobuffer_advice;
```

Comment:
I think what we ran into last time was a dependency ordering issue, which is why we scrapped the whole idea of a config variable in the first iteration.